README.md

```diff
@@ -1,120 +1,15 @@
-# AWS S3 bucket Terraform module
```

Comment:
Please revert all these docs changes in README, I will fix the rest of this PR (it is too confusing, I know).

Reply:
This has not been done yet.

Reply:
Is it possible to explain what's expected this way I can make sure I generate the right documentation ? 
I am running terraform-docs 0.13.0 from a docker container locally before pushing.
Sorry for the back and forth.

Reply:
In this PR you deleted most of the content of the README. Please revert that one, and then running `pre-commit run -a` will fix the required part related to Terraform versions (if not, I can fix it for you).

Reply:
Oh got it now. Normally this should be fixed in the last push.